client/extensions/woocommerce/state/ui/shipping/zones/test/selectors.js

```diff
@@ -41,12 +41,42 @@ describe( 'selectors', () => {
 			const state = {
```

Comment:
You could also create a `fixtures` folder inside the `test` folder to hold any "larger" mock data (see [the orders state fixtures in this branch](https://github.com/Automattic/wp-calypso/tree/add/woocommerce-orders-state/client/extensions/woocommerce/state/sites/orders/test/fixtures))